app/graphql/types/user_type.rb

```diff
@@ -36,6 +35,10 @@ def weekly_habits
       object.weekly_habits
     end
 
+    def journal_entries
+      object.journal_entries.weekly_journal
```

Comment:
You could abstract this one level deeper so you don't need to reach into the objects(users) journal entries
User Model: 
`scope :weekly_journals, -> {
    journal_entries.weekly_journal
}`
 User Type:
`def journals
     object.weekly_journals
end`

Reply:
Updated as an instance method